network/topics/params/helpers.go

```diff
@@ -7,10 +7,10 @@ import (
 	"github.com/pkg/errors"
 )
 
-// scoreByWeight provides the relevant score by the provided weight and threshold.
-func scoreByWeight(maxScore float64, weight, threshold float64) float64 {
-	return maxScore / (weight * threshold * threshold)
-}
+const (
+	oneEpochDuration = (12 * time.Second) * 32
+	slotsPerEpoch    = 32
+)
```

Comment:
we better use values from the network config, but we'll have to pass them around to do that.

Reply:
`slotsPerEpoch` was unused, so I removed it.
`oneEpochDuration` is used as the default value if the `Options.Network.OneEpochDuration` is not set.